### ISSUE TITLE
code for shape-consistency loss

### ISSUE THREAD

**Issue by xianzhongma** [YadiraF/DECA#22]
I understand the idea behind the shape-consistency loss and think it is reasonable. But I feel confused how to write the code. According to the paper, shape-consistency loss is the loss of exchanging the beta for two example of the same identity. However, batch size is usually greater than 2， so how to deal with a batch of examples? Any suggestions and comments are welcomed. Thanks.

**Comment by TimoBolkart** [comment]
The training code is not available, so please find the implementation details in the code

**Comment by hillaric** [comment]
can you speak detail about the consistency loss? In the trainer.py (L 125), I can not understand how you calculate the loss?

**Comment by bxiong97** [comment]
>The training code is not available

Do you mean it is **now** available?
